site/gatsby-site/server/tests/emailRateLimit.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { RateLimiter } from "limiter";
+import * as emails from "../emails";
+import { EmailParams } from "mailersend";
+
+const sendBulkMock = jest.fn().mockResolvedValue(true);
+
+jest.mock("mailersend", () => ({
+    MailerSend: jest.fn().mockImplementation(() => ({
+        email: {
+            sendBulk: sendBulkMock,
+        },
+    })),
+}));
+
+describe("mailersendBulkSend with rate limiter", () => {
+
+    it("Allows only 10 calls per minute", async () => {
+
+        emails.setLimiter(new RateLimiter({
+            tokensPerInterval: 10,
+            interval: 2000,
+        }));
```

Comment:
Do we need to set this limiter here as well?

Reply:
It is so we don't have to wait a minute for the test to finish.